FormidableLabs/spectacle
# Revamped `create-spectacle` CLI and getting started documentation for Spectacle.

1. Updated `create-spectacle` CLI to have new Markdown option, make One Page first class, and remove older JS-based options.

<img width="692" alt="Screenshot 2023-12-15 at 9 10 55 AM" src="https://github.com/FormidableLabs/spectacle/assets/1738349/29771820-45ce-4fcb-bc57-8f04e5b0cad6">

2. Revamped Spectacle documentation to remove old installation methods and show a cleaner getting started guide using the new CLI → https://spectacle-docs-website-git-feature-creat-c38d7e-formidable-labs.vercel.app/docs/


**How to test**

1. cd into `packages/create-spectacle` and run `pnpm dev`
2. run through each option
3. cd into the generated directories and run `npm i && npm start` to verify each output type works
4. for one-page just double click the html file and see that it loads

For release this will get folded into this changeset for the CLI → https://github.com/FormidableLabs/spectacle/pull/1305
